From 99f0dad49ca71853371c041b2d13f9cca8b7f9f3 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Wed, 29 Jan 2025 12:40:39 +0200
Subject: [PATCH] [SW-216694] common: adjust header include locations

Include headers are moved under include/zephyr with zephyr
3.7.1, so adjust in all files along with new header additions
whereever required.

Change-Id: Ib91ec17ea53c3e3f43a74652f10df5de50b62655
---

diff --git a/arch/fs1/palad.c b/arch/fs1/palad.c
index 38e9b21..0a27ac6 100644
--- a/arch/fs1/palad.c
+++ b/arch/fs1/palad.c
@@ -15,7 +15,7 @@
 #ifdef __ZEPHYR__
 #include <preboot.h>
 #include <hl_device.h>
-#include <drivers/misc/common/common_utils.h>
+#include <zephyr/drivers/misc/common/common_utils.h>
 #include <hl_platform_defs.h>
 #else
 #include <baremetal.h>
